code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -4374,3 +4373,258 @@
 	New()
 		..()
 		reagents.add_reagent(NUTRIMENT,3)
+
+/obj/item/weapon/reagent_containers/food/snacks/threebeanburrito
+	name = "three bean burrito"
+	desc = "Beans, beans a magical fruit."
+	icon_state = "danburrito"
+	bitesize = 2
+	New()
+		..()
+		reagents.add_reagent(NUTRIMENT,5)
+
+/obj/item/weapon/reagent_containers/food/snacks/midnightsnack
+	name = "midnight snack"
+	desc = "Perfect for those occasions when engineering doesn't set up power."
+	icon_state = "midnightsnack"
+	bitesize = 2
+	New()
```

Comment:
https://github.com/d3athrow/vgstation13/pull/12852  did it 4 days ago and believe me it was exactly as tedious as you think it was.